package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Kinase is an extension for Google Chrome allowing users to map web content to a list of annotations.",
   "author": "B12 <hello@b12.io> (https://b12.io/)",
   "scripts": {
-    "release": "node scripts/release.js",
-    "test": "webpack"
+    "prepublish": "webpack",
+    "release": "np"
```

Comment:
Originally took this out because it'll run twice on publish, but I think you're definitely right ✅